apps/prairielearn/src/admin_queries/course_instances_usage.json

```diff
@@ -0,0 +1,20 @@
+{
+  "description": "Course instances usage data.",
```

Comment:
This is the admin query that we will normally run for a single institution, with the dates set to one term.